pom.xml

```diff
@@ -65,7 +65,7 @@
     -->
         <repository>
             <id>maven-restlet</id>
-            <url>https://maven.restlet.org</url>
+            <url>https://maven.restlet.talend.com</url>
```

Comment:
Interesting change. Fragile repo maybe?

Reply:
https://maven.restlet.org actually redirects to https://maven.restlet.talend.com/

I changed this because I was getting a warning about restlets, and this seemed to fix it, but now I can't reproduce when I revert.

See this comment immediately before it:
```xml
    <!--
        2021-09-24 <mecum@nceas.ucsb.edu>
        Remove this eventually: http->https mirror for maven.restlet that we can
        remove once we update solr-core to a version that doesn't have the http
        repository for restlet.
    -->
```
Actually, deleting this whole repo definition doesn't seem to break anything, so maybe I should just remove the whole thing??

Reply:
@taojing2002 what do you think?

Reply:
maybe use `maven deps` and see what uses it?

Reply:
summary from slack convo with Jing:

> 
> ```
> # (rm -rf ~/.m2/repository; pom repo definition intact...)
> $ mvn dependency:analyze > MVN_DEP_WITH_RESTLET_REPO.txt
> 
> # (rm -rf ~/.m2/repository again; commented out repo definition, then...)
> $ mvn dependency:analyze > MVN_DEP_NO_RESTLET_REPO.txt
> 
> $ diff -bBdry --suppress-common-lines MVN_DEP_NO_RESTLET_REPO.txt MVN_DEP_WITH_RESTLET_REPO.txt
> 
> [INFO] Total time: 04:18 min				  | [INFO] Total time: 04:27 min
> [INFO] Finished at: 2023-11-02T09:42:44-07:00 | [INFO] Finished at: 2023-11-02T09:50:51-07:00
> ```
> i.e. no real difference. Deleting!